desktop/src/main/java/bisq/desktop/main/support/dispute/agent/SignedOfferView.java

```diff
@@ -0,0 +1,425 @@
+                           TradeManager tradeManager, OfferDetailsWindow offerDetailsWindow,
+                           CoreApi coreApi) {
+        this.openOfferManager = openOfferManager;
+        this.xmrWalletService = xmrWalletService;
+        this.tradeManager = tradeManager;
+        this.offerDetailsWindow = offerDetailsWindow;
+        this.coreApi = coreApi;
+    }
+
+    private Offer fetchOffer(SignedOffer signedOffer) {
+        Trade trade = tradeManager.getTrade(signedOffer.getOfferId());
+
+        return trade != null ? trade.getOffer() : null;
+    }
+
+    ///////////////////////////////////////////////////////////////////////////////////////////
+    // Life cycle
+    ///////////////////////////////////////////////////////////////////////////////////////////
+
+    @Override
+    public void initialize() {
+        Label label = new AutoTooltipLabel(Res.get("support.filter"));
+        HBox.setMargin(label, new Insets(5, 0, 0, 0));
+        HBox.setHgrow(label, Priority.NEVER);
+
+        filterTextField = new InputTextField();
+        Tooltip tooltip = new Tooltip();
+        tooltip.setShowDelay(Duration.millis(100));
+        tooltip.setShowDuration(Duration.seconds(10));
+        filterTextField.setTooltip(tooltip);
+        HBox.setHgrow(filterTextField, Priority.NEVER);
+
+        filterTextField.setText("open");
+
+        setupTable();
+    }
+    @Override
+    protected void activate() {
+        super.activate();
+
+        ObservableList<SignedOffer> observableList = FXCollections.observableArrayList();
+        observableList.addAll(openOfferManager.getSignedOffers());
+
+        SortedList<SignedOffer> sortedList = new SortedList<>(observableList);
+        sortedList.comparatorProperty().bind(tableView.comparatorProperty());
+        sortedList.addListener((ListChangeListener<SignedOffer>) c -> tableView.refresh());
+        tableView.setItems(sortedList);
+
+        contextMenu = new ContextMenu();
+        MenuItem item1 = new MenuItem(Res.get("support.contextmenu.penalize.msg",
+                Res.get("shared.maker")));
+        contextMenu.getItems().addAll(item1);
+
+        tableView.setRowFactory(tv -> {
+            TableRow<SignedOffer> row = new TableRow<>();
+            row.setOnContextMenuRequested(event -> {
+                contextMenu.show(row, event.getScreenX(), event.getScreenY());
+            });
+            return row;
+        });
+
+        item1.setOnAction(event -> {
+            selectedSignedOffer = tableView.getSelectionModel().getSelectedItem();
+            Offer offer = fetchOffer(selectedSignedOffer);
+            log.debug("Found {} matching signed offer.", (offer != null ? offer.getId() : null));
+            if(offer != null) {
+                Transaction tx = null;
+                if(offer.getOfferFeePaymentTxId() != null) {
+                    tx = coreApi.getTransaction(offer.getOfferFeePaymentTxId());//Digging up the mining fee
```

Comment:
`coreApi.getTransaction()` only applies to BTC transactions, so this would never work, and you clearly are not testing your code before submitting for review.

Submissions are expected to be tested before review. Otherwise it unnecessarily costs me time. Please test all flows first.

This call will be removed by only using fields available in `SignedOffer`.

Reply:
This was committed because I didn't want to lose the work I was doing (stash wouldn't do the trick). Laptop was likely going to crash. Sorry for not flagging it and sending you in the wrong direction.

Yes, several of what you noted will be reverted. Thanks for keeping up.

Reply:
This has been long since removed.